DEMO/org.openl.rules.demo/jetty-conf/demo-java.policy

```diff
@@ -1,240 +1,215 @@
-
-        //Warning: Extreme caution should be taken before granting this permission to code, for it provides access to the declared members of a class.
-        //ex: springframework.beans.factory
-        permission java.lang.RuntimePermission "accessDeclaredMembers";
-
-        //Reading of the value of the specified environment variable
-        permission java.lang.RuntimePermission "getenv.*";
-
-        //access to the java.util.prefs.Preferences implementations
-        permission java.lang.RuntimePermission "preferences";
-
-        permission java.lang.RuntimePermission "reflectionFactoryAccess";
-
-        //allows anyone to listen on dynamic ports (DB, LDAP, SSO...)
-        permission java.net.SocketPermission "*", "accept,connect,listen,resolve";
-
-        // Read and Modify any properties
-        permission java.util.PropertyPermission "*", "read,write";
-
-        //ability to control the logging configuration
-        permission java.util.logging.LoggingPermission "control";
-
-        permission java.lang.reflect.ReflectPermission "*";
-
-        //FileUtils
-        permission java.io.FilePermission "${java.io.tmpdir}", "read,write,delete";
-        permission java.io.FilePermission "${java.io.tmpdir}/-", "read,write,delete";
-
-        permission java.io.FilePermission "${openl.home}", "read,write,delete";
-        permission java.io.FilePermission "${openl.home}/-", "read,write,delete";
-
-        //org.springframework.beans.factory
-        permission java.io.FilePermission "${catalina.base}", "read";
-        permission java.io.FilePermission "${catalina.base}/-", "read";
-
-        permission java.io.FilePermission "${catalina.base}/logs/-", "write,delete";
-        permission java.io.FilePermission "${java.home}/-", "read";
-        permission java.util.PropertyPermission "*", "read,write";
-
-        //oracle.jdbc.driver.OracleDriver.registerMBeans
-        permission javax.management.MBeanServerPermission "createMBeanServer";
-        permission javax.management.MBeanPermission "*", "*";
-        permission javax.management.MBeanTrustPermission "register";
-
-        //tomcat
-        permission java.net.NetPermission "specifyStreamHandler";
-
-        //log4j
-        permission java.lang.RuntimePermission "getStackWalkerWithClassReference";
-
-        //openapi
-        permission java.lang.management.ManagementPermission "monitor";
-        permission java.lang.RuntimePermission "createSecurityManager";
-        permission java.lang.RuntimePermission "shutdownHooks";
-
-        //ZipFileSystem for JDK 8 issue
-        permission java.io.FilePermission "${catalina.base}/lib/Example3-datasource.jar", "read,write";
-
-        // "/groovy/script" is used in GroovyClassLoader parseClass method
-        permission groovy.security.GroovyCodeSourcePermission "/groovy/script";
-};
+// These permissions apply to the server startup code
+grant codeBase "file:${jetty.home}/start.jar" {
```

Comment:
Please, keep the same line breaks. If previously was used CRLF, don't change it to LF and vice versa